DownloadFiles.md

```diff
@@ -0,0 +1,79 @@
+# Download Document Files
+Extensions can download the files associated with documents.
+
+A UIX can identify the file to download in the following ways:
+- Download a file based on the file ID (9.6.002)
+- Download a content file based on the content file type ranking (9.6.002)
+- Download a page file based on the page file number (9.6.002)
+- Download a page file as a .jpg or .png image (9.7.000)
```

Comment:
Minor point, the image download API downloads images of pages, not page files. Page "files" can be PDFs with many pages inside, but the image download API would fetch images of just one of it's pages.

Reply:
I fixed this and the other stuff. Have another look.